driver/Cargo.toml

```diff
@@ -15,6 +15,8 @@ futures = { version = "0.3.5", features = ["compat"] }
 isahc = { version = "0.9.2", features = ["json"] }
 lazy_static = "1.4.0"
 log = "0.4.8"
+num-bigint = { version = "0.2", features = ["serde"] }
+num-traits = "0.2"
```

Comment:
ubernit: I would just use the `num` crate, its fairly small and won't add much compilation time. It would also make dependabot's life easier at it has a hard time upgrading crates whose versions are tied together and I think will likely require manual intervention.